packages/opds-web-client/src/components/Root.tsx

```diff
@@ -409,7 +409,7 @@ export class Root extends React.Component<RootProps, RootState> {
   }
 }
 
-let connectOptions = { withRef: true, pure: false };
+let connectOptions = { pure: false };
```

Comment:
@EdwinGuzman I think you understand this better than I do from your previous updates. I feel like this may have already stopped working in the previous update but it didn't get caught before because we didn't have the new Redux typings. You apparently can use `forwardRef: true` here instead, but I'm not sure if we need that.

Reply:
I see. Adriana is working on an update for the form in the admins and it doesn't seem to need the `forwardRef` at all but this is good to know.